woocommerce/woocommerce-android
# Issue/8277 move ipp learn more link to payments menu

Closes: #8277 


### Description

This PR removed the` Learn More` view from the `Scanning Readers` modal and adds it to the Payments Hub screen

### Testing instructions


1. Go to Menus
2. Go to Payments
3. Check if the learn more view shows underneath the rows
4. test on landscape mode, make sure it does not disappear 
5. enable and disable Pay in Person option, make sure the Learn More view is displayed in both states
6. Click on the Learn More option
7. make sure it takes you to the right webview
8. make sure the event is tracked in the logs
9. Back to the payments menu create a simple payment
10. select collect payment
11. Make sure the learn more link also shows there
12. select the card option  (make sure you don't have any readers connected so the modal stays in the screen)
13. make sure the learn more view is not displayed in the modal anymore. 

### Images/gif

<img src="https://user-images.githubusercontent.com/30724184/216646860-aa0a9c74-f644-4907-ab02-f64420f05b4f.png" width="350"/> 

<img src="https://user-images.githubusercontent.com/30724184/218869677-9a28dc09-ddb1-4075-8f65-e4f055d33c05.png" width="350"/> 

<img width="333" alt="Screen Shot 2023-02-14 at 10 38 33 PM" src="https://user-images.githubusercontent.com/30724184/218869773-3fd5cc8a-af7a-461a-81e6-bcf3823ff241.png">

<img width="307" alt="methods" src="https://user-images.githubusercontent.com/30724184/218869825-fa693ab4-ed68-4758-95ba-90f66f6eb8bc.png">


<img src="https://user-images.githubusercontent.com/30724184/216646934-d2b31e16-5c82-4641-9355-01def857e0f4.png" width="350"/>
